### DISCUSSION TITLE
article/vercel-deploy-notion-next

### DISCUSSION THREAD

**Discussion by giscus** [tangly1024/tangly1024#56] [Announcements]
# article/vercel-deploy-notion-next

Vercel是一款国外的Serverless托管平台，对个人用户使用几乎免费，而且方便快捷，用Vercel托管你的Notion站点，无需再操心服务器的维护与资费问题。

https://tangly1024.com/article/vercel-deploy-notion-next

**Comment by fulaoaz** [comment]
步骤三的第二步加载不出来

**Reply by alexlabxs** [reply]
vercel为您的站点提供一个 免费的*.vercel.app 域名，但vercel的官方域名在大陆被墙，推荐您绑定自己的域名。

**Comment by lee-forever** [comment]
求助，用github账号登陆vercel，vercel显示Account not found。重新注册了个github账号登陆还是一样。

**Reply by lee-forever** [reply]
先注册了个vercel账号，然后再continue with GitHub，终于成功了，谢谢教程

**Comment by xweier** [comment]
始终无法同步blog.config.js 里面的内容，做了修改vercel始终提示部署失败。

**Reply by lee-forever** [reply]
你有没有用github汉化脚本？

**Comment by xweier** [comment]
没有用。

获取 Outlook for iOS<https://aka.ms/o0ukef>
________________________________
发件人: lee-forever ***@***.***>
发送时间: Sunday, July 16, 2023 6:23:01 PM
收件人: tangly1024/tangly1024 ***@***.***>
抄送: RASHweier ***@***.***>; Comment ***@***.***>
主题: Re: [tangly1024/tangly1024] article/vercel-deploy-notion-next (Discussion #56)


你有没有用github汉化脚本？

―

**Comment by muzituandui** [comment]
ID也没填错。就是显示无法获取内容 咋整

**Comment by zsr-lukezhang** [comment]
我在这里提示.vercel.app的使用者：
*.vercel.app在大陆被墙。请你使用你自己的域名。然后将域名托管至Cloudflare。把DNS记录Proxy选项打开（默认就打开了）
然后应该就可以正常访问了。实在不行换个部署方式就行了。

**Reply by zsr-lukezhang** [reply]
顺便说一下，在不在Cloudflare买的没关系。重要的是要把DNS名称服务器改为Cloudflare的。
我现在看到的最便宜的域名实在Cloudflare卖的*.win域名。每年差不多￥30。
